transaction_integration_test.go

```diff
@@ -990,6 +990,9 @@ func TestAllTransactionFields(t *testing.T) {
 	if tx2.SubscriptionDetails != nil {
 		t.Fatalf("expected Subscription to be not nil, but got %#v", tx2.SubscriptionDetails)
 	}
+	if tx2.AuthorizationExpiresAt == nil {
```

Comment:
Could we test that the value isn't only not `nil` but that it also has a value in a range of time that would be appropriate?